src/actions.sol

```diff
@@ -94,6 +98,11 @@ contract Actions is DSNote {
         shelf.repay(loan, amount);
     }
 
+    function repayFullDebt(ShelfLike shelf, PileLike pile, ERC20Like erc20, uint loan) public {
```

Comment:
Can we add always the loan id as the first parameter to be consistent with the core contracts